modules/standard/IO.chpl

```diff
@@ -4099,7 +4144,17 @@ proc channel.readBinary(ref arg:numeric, param endian:ioendian = ioendian.native
   return true;
 }
 
-proc channel.readBinary(ref arg:numeric, endian: ioendian):bool throws {
+/*
+   Read a binary number from the channel
+
+   :arg arg: number to be read
+   :arg endian: type:`ioendian` specifies the byte order in which
+              to read the number. Defaults to `ioendian.native`.
```

Comment:
Ditto here about the `:type:` missing colon and missing ` around ioendian.native